ios-swiftui/realm-template-apps-swiftui/Views/NewTodo.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import RealmSwift
+
+/// Create a new task.
+struct NewTodo: View {
+    // Appending objects to the ObservedResults collection
+    // adds them to the realm.
+    @ObservedResults(Todo.self) var todos
+    @State private var newTodo = Todo()
+    @Binding var addTodo: Bool
```

Comment:
Is this the best name for this variable or is it more about the state of whether the view is done adding the todo (or not adding the todo due to canceling) - that is the open state of the view?